extensions/ql-vscode/src/run-queries.ts

```diff
@@ -186,9 +195,21 @@ export class QueryEvaluationInfo {
       if (queryInfo && await qs.cliServer.cliConstraints.supportsPerQueryEvalLog()) {
         await qs.sendRequest(messages.endLog, {
           db: dataset,
-          logPath: this.structLogPath,
+          logPath: this.evalLogPath,
         });
-        queryInfo.evalLogLocation = this.structLogPath;
+        if (await this.hasEvalLog()) {
+          queryInfo.evalLogLocation = this.evalLogPath;
+          await qs.cliServer.generateLogSummary(this.evalLogPath, this.evalLogSummaryPath, this.evalLogEndSummaryPath);
+          fs.readFile(this.evalLogEndSummaryPath, (err, buffer) => {
+            if (err) {
+              throw new Error(`Could not read structured evaluator log end of summary file at ${this.evalLogEndSummaryPath}.`);
+            }
+            void qs.logger.log(' --- Evaluator Log Summary --- ', { additionalLogLocation: this.logPath });
```

Comment:
I _think_ this will add the `' --- Evaluator Log Summary --- '` to the file. Is this what we want? Is it better/possible to ensure the output is proper json or jsonl?

Reply:
This adds the ' --- Evaluator Log Summary --- ' line to the Query Server Console, which I believe is what we want. The file itself (the end summary file) should never be read by a user; it's generated on the CLI side specifically for this use case.

Reply:
The additionalLogLocation will add the contents to `this.logPath`. If it's not meant to be read by a user and it is not jsonl syntax then adding this line to it as well is fine.

Reply:
Hm, the user is meant to read this line (and the following lines) only from the Query Server Console. I can remove the `additionalLogLocation` part of the logging so that the user will not see traces of the evaluator structured logs in the additional log where they may not expect it. If they want the full summary of the structured log, they can view the actual summary file.

Reply:
You are much more familiar with this feature than I am. I am fine with it if you decide to leave this in.